deploy.sh

```diff
@@ -1,5 +1,30 @@
 #!/bin/bash
 
+# Node should be installed
+
+if (which node)
+then 
+ echo "node is installed"
+else
+ echo "installing node"
+ brew install node
+fi
+
+# https://github.com/nodesource/distributions#deb
+
+
 # Npm should be installed
+# gets installed with brew install node
+
+if (npm -version)
+then 
+ echo "npm is installed"
+else
+ echo "installing npm"
+ brew upgrade node
+fi
+
+# npm install
 
-if (npm version)
```

Comment:
-version is not verbose, if we are only checking for desired state then verbose is not needed.